src/sections/Community/Handbook/repository.js

```diff
@@ -19,11 +26,42 @@ const Repository = () => {
       <div className="page-section">
         <Container>
           <div className="content">
-            <h2>Purpose</h2>
+            <h2>GitHub Organizations and Repositories</h2>
             <p>
-              This overview serves as resources to newcomers seeking a
+              This overview serves as a resource to newcomers seeking a
               perspective of the collective community efforts (from the limited
               vantage point of a code-centric perspective).
+              <br /> Note that the Layer5 community spans four GitHub
+              organizations:
+            </p>
+            <p>
+              <img
```

Comment:
![repooverview](https://user-images.githubusercontent.com/85789734/135705578-494ba213-70c3-4ea3-bfff-a183e579e939.jpg)
These logos look very out of place. After mentioning` 4 GitHub organizations: ` Just list the 4 organizations one by one and add their GitHub links.

Reply:
Basically make an unordered list, instead of bullets, add the respective logos for the organizations and the names which will be linked to the orgs. That should be enough for this section.